### ISSUE TITLE
[Accessibility][Labels or Instructions - Create from scratch]: 'Create from Scratch/Template' labels are not associated with its radio buttons.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2018]
**User Experience:**
Screen reader users will not get proper information and may get difficulty in performing the task efficiently if Labels are not associated with there respective radio buttons.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**

1. Open URL: http://localhost:3000/home
2. Navigate on homepage with tab key up to 'New' button and press enter on it.
3. 'Create from scratch' pop up will open.
4. Observe the issue on radio buttons.

**Actual Result:**
'Create from Scratch/Template' labels are not associated with its radio buttons. Screen reader only read as 'Radio button selected 1 of 2 required'.

**Note:**
Similar issue is observed for: Notifications (from left navigation bar) > Page. Page label is not associated with the page count. Please: Refer: Attachment.

**Expected Result:**
'Create from Scratch/Template' labels should be associated with its radio buttons. Screen reader should read as 'Create from Scratch/Template Radio button selected 1 of 2 required' so that user will  get to know the complete information that what screen reader is reading.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachment:**

[MAS3.3.2_Create from Scratch&Template radio button_labels_K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4224642/MAS3.3.2_Create.from.Scratch.Template.radio.button_labels_K4W.zip)
[MAS3.3.2_Create from Scratch&Template radio button_labels_Screen reader.zip](https://github.com/microsoft/BotFramework-Composer/files/4224643/MAS3.3.2_Create.from.Scratch.Template.radio.button_labels_Screen.reader.zip)
[MAS3.3.2_Create from Scratch&Template radio button labels_K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4224645/MAS3.3.2_Create.from.Scratch.Template.radio.button.labels_K4W.zip)

[Notification_PageLabel_K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4266241/Notification_PageLabel_K4W.zip)

[MAS3.3.2_Notification_Page Label Video (1).zip](https://github.com/microsoft/BotFramework-Composer/files/4266248/MAS3.3.2_Notification_Page.Label.Video.1.zip)
[MAS3.3.2_Notification_Page Label.zip](https://github.com/microsoft/BotFramework-Composer/files/4266250/MAS3.3.2_Notification_Page.Label.zip)

**Comment by corinagum** [comment]
![MAS3 3 2_Create from Scratch Template radio button_labels_K4W](https://user-images.githubusercontent.com/14900841/75414298-ec603980-58dc-11ea-8bc9-9b822e0865f5.png)


Only .png in .zip, pasted above; no video file requiring download :)

**Comment by awalia13** [comment]
Hi Corina ,We will validate this bug and share update asap.

**Comment by ashish315** [comment]
@carinagum As checked still the issue didn't resolved. Still we getting same issue by Keros.

**Comment by corinagum** [comment]
Please re-verify. I have confirmed the fix on master branch of Composer.

**Comment by ashish315** [comment]
As checked on the latest build downloaded on 09-04-2020 2:00 AM
The issue is fixed at Notification pane but still issue repro's on 'Create from scratch'. PFA

**Comment by ashish315** [comment]
As checked, the issue is resolved. So, HCL is closing the issue.
